.travis.yml

```diff
@@ -6,7 +6,8 @@ services:
 install:
   - docker build -t max-image-caption-generator .
   - docker run -it -d -p 5000:5000 max-image-caption-generator
+  - sleep 30
```

Comment:
Without some sort of pause here the travis will get to the actual test step before the api server finishes starting, I went with `30` since it would always be long enough, even if `10` is the average start time.

Reply:
ok fair enough